sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReleaseInfo.java

```diff
@@ -49,11 +49,16 @@ public String getName() {
     return getProperties().get("name");
   }
 
-  /** Provides the SDK version. */
+  /** Provides the BEAM version. */
```

Comment:
It might be good to provide examples here to explain the difference.

Reply:
good idea, added it.